src/models/Division.ts

```diff
@@ -0,0 +1,6 @@
+import { Team } from './Team';
```

Comment:
I'm a little surprised this is building? There is no folder called `src/models` and thus no `src/models/Team.ts` in `main`. You might want to do a `git pull origin main` in this branch 🙂 

Anyway, not a big deal, but in my last big boi PR, I shuffled things around to a more domain-driven folder structure. We should find a spot for these in the new structure. They'll probably end up somewhere like `src/components/division/state/division-model.ts` and `src/components/game/state/game-model.ts`